### DISCUSSION TITLE
How To Set Up Navigation Within Form?

### DISCUSSION THREAD

**Discussion by evelynDickinson** [OpnForm/OpnForm#270] [Ideas & Feature Requests]
Hey there. 

I'm building a 20-ish page form with multiple sections, and I'm wondering if it's possible to use code blocks to set up navigation within the form to jump between sections on different pages like old-school anchor tags, or is there a better way?

&lt;h1 id="section-1"&gt;Section 1&lt;/h1&gt;

&lt;a href="#section-1"&gt;Go To Section 1&lt;/a&gt;

OpnForm is awesome. Thank you for all the hard work!

**Comment by JhumanJ** [comment]
Hey @evelynDickinson, you could use code blocks and anchor links, bu it would not work to jump from one page to another if you're using page breaks unfortunately.

**Comment by kdvan-hub** [comment]
Can you make the logo on upper left clickable so we can add a URL link back to the homepage?

**Reply by JhumanJ** [reply]
Can you share a screenshot? OpnForm logo is currently clickable

**Comment by kdvan-hub** [comment]
Hi Julien,

Here is a my form screenshot and I'm not able to make my logo clickable.
[image: Screenshot 2024-06-16 090230.png]

**Reply by JhumanJ** [reply]
Not seeing anything :(
